CHANGELOG.md

```diff
@@ -60,6 +60,7 @@
 * [BUGFIX] Querier: reduce log volume when querying ingesters with zone-awareness enabled and one or more instances in a single zone unavailable. #6381
 * [BUGFIX] Querier: don't try to query further ingesters if ingester query request minimization is enabled and a query limit is reached as a result of the responses from the initial set of ingesters. #6402
 * [BUGFIX] Ingester: Don't cache context cancellation error when querying. #6446
+* [BUGFIX] Ingester: don't ignore errors encountered while iterating through chunks or samples in response to a query request. #6469
```

Comment:
Note to reviewers: this changelog line is the same as for https://github.com/grafana/mimir/pull/6451 - both fix the same kind of issue, and I don't think the differences between them matter to a user reading the changelog.